charts/fission-workflows/templates/deployment.yaml

```diff
@@ -76,9 +79,26 @@ metadata:
   namespace: default
 spec:
   version: 2
+  TerminationGracePeriod: {{ .Values.gracePeriod }}
   runtime:
     image: "{{ .Values.envImage }}:{{.Values.tag}}"
     container:
+      command: ["/fission-workflows-bundle"]
+      args: [
+      # Only run components related to accessing the API
```

Comment:
This comment is not true for the workflow environment; could you instead replace it with a description why we override the settings of https://github.com/fission/fission-workflows/blob/master/build/runtime-env/Dockerfile#L21 ?
Something like `# Override workflow env CMD to optionally enable --debug `